### ISSUE TITLE
PlanR1.py里面强化学习部分的疑问

### ISSUE THREAD

**Issue by 123sdadaw** [XiaolongTang23/Plan-R1#11]
作者你好！
model\PlanR1.py，第162行：
ratio = (plan_logps - plan_logps.detach()).exp()
是否应该改为下面的啊：
ratio = (plan_logps - pred_logps.detach()).exp()
